CollAction/Frontend/app/global/RichTextEditor.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
-import * as ReactQuill from "react-quill";
 import renderComponentIf from "./renderComponentIf";
+const ReactQuill = require("react-quill");
```

Comment:
What's wrong with the import statement? I'm not really familiar with this 'const' construct...